.drone.yml

```diff
@@ -82,6 +82,7 @@ steps:
     image: plugins/docker
     when:
       branch:
+        - staging
         - prod
```

Comment:
Okay so the plan here is to merge everything into the `staging` branch first? how agile 😨

Reply:
weeell I mostly just want it to be an option..

as I see it the options are either building it in master, or building it in both staging and prod. And with the frequency of the builds of this project it seems unnecessary to build for every push to master (mostly dependabot).  
Could of course make a build branch and do the building there - but it seems a bit impractical to merge to main, push to build, push to staging, push to prod and have to wait for the steps to complete for each change.

Leaves the problem that occured in kvittering where dev/staging is miles behind prod and master though, so it might be better to merge everything through master..

Reply:
Simple enough to change at a later time though. For now I think this'll work all right and I'll see if I can think of a more effective way to 1. make sure staging doesn't get lost behind 2. avoid running CI unnecessarily for every new commit to master 3. have staging up and running later